Snowplow.Analytics.Tests/Json/EventTransformerTest.cs

```diff
@@ -632,6 +632,200 @@ private string GetSerializedExpectedOutputForInputWithoutContextAndUnstructEvent
+                'mkt_term' : null,
+                'mkt_content' : null,
+                'mkt_campaign' : null,
+                'contexts_org_schema_web_page_1' : [ {
+                  'genre' : 'blog',
+                  'inLanguage' : 'en-US',
+                  'datePublished' : '2014-11-06T00:00:00Z',
+                  'author' : 'Devesh Shetty',
+                  'breadcrumb' : ['blog', 'releases'],
+                      'keywords' : [ 'snowplow', 'javascript', 'tracker', 'event' ]
+                } ],
+                'contexts_org_w3_performance_timing_1' : [ {
+                  'navigationStart' : 1415358089861,
+                  'unloadEventStart' : 1415358090270,
+                  'unloadEventEnd' : 1415358090287,
+                  'redirectStart' : 0,
+                  'redirectEnd' : 0,
+                  'fetchStart' : 1415358089870,
+                  'domainLookupStart' : 1415358090102,
+                  'domainLookupEnd' : 1415358090102,
+                  'connectStart' : 1415358090103,
+                  'connectEnd' : 1415358090183,
+                  'requestStart' : 1415358090183,
+                  'responseStart' : 1415358090265,
+                  'responseEnd' : 1415358090265,
+                  'domLoading' : 1415358090270,
+                  'domInteractive' : 1415358090886,
+                  'domContentLoadedEventStart' : 1415358090968,
+                  'domContentLoadedEventEnd' : 1415358091309,
+                  'domComplete' : 0,
+                  'loadEventStart' : 0,
+                  'loadEventEnd' : 0
+                } ],
+                'se_category' : null,
+                'se_action' : null,
+                'se_label' : null,
+                'se_property' : null,
+                'se_value' : null,
+                'unstruct_event_com_snowplowanalytics_snowplow_link_click_1' : {
+                  'targetUrl' : 'http://www.example.com',
+                  'elementClasses' : [ 'foreground' ],
+                  'elementId' : 'exampleLink'
+                },
+                'tr_orderid' : null,
+                'tr_affiliation' : null,
+                'tr_total' : null,
+                'tr_tax' : null,
+                'tr_shipping' : null,
+                'tr_city' : null,
+                'tr_state' : null,
+                'tr_country' : null,
+                'ti_orderid' : null,
+                'ti_sku' : null,
+                'ti_name' : null,
+                'ti_category' : null,
+                'ti_price' : null,
+                'ti_quantity' : null,
+                'pp_xoffset_min' : null,
+                'pp_xoffset_max' : null,
+                'pp_yoffset_min' : null,
+                'pp_yoffset_max' : null,
+                'useragent' : 'Mozilla/5.0 (Linux; Android 9; SM-G950U Build/PPR1.180610.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/94.0.4606.61 Mobile Safari/537.36 (""Affirm - Android - 3.112.3 - 509\"")',
```

Comment:
I'm surprised that this is the desired output string. Why doesn't the first set of `""` have a `\`? And why are there two `"` instead of one?

Reply:
The backslash was ignored in the string comparison that's why the test passed, but good point, it shouldn't have been there. Have removed it. The double quotes are just a way to escape them.